exetera/core/validation.py

```diff
@@ -287,3 +287,31 @@ def validate_and_normalize_categorical_key(param_name, key):
             return {k: v.encode() for k, v, in key}
         else:
             return key
+
+
+def validate_sort_parameter(axis, ascending, kind):
```

Comment:
This validation is super-specific to sorting; in general, functions are put in validation.py if they have general applicability.